api/swagger-spec/swagger.yml

```diff
@@ -275,6 +301,13 @@ definitions:
         type: array
         items:
           $ref: "#/definitions/componentVersion"
+  doctorInfo:
+    type: object
+    required:
+      - cluster
+    properties:
+      cluster:
+        $ref: "#/definitions/cluster"
```

Comment:
Yes 
Although I haven;t wrote and get route but its the same and display doctor data struct. fair enough ?